code/modules/economy/commodity.dm

```diff
@@ -573,6 +573,12 @@
 ///// syndicate trader /////////
 ////////////////////////////////
 
+/datum/commodity/clothing_restock
+	comname = "Clothing Vendor Restock Cartridge"
+	desc = "A restock cartridge for restocking clothing vending machines."
+	comtype = /obj/item/vending/restock_cartridge/jobclothing
+	price = PAY_IMPORTANT
```

Comment:
This should just be for the syndie vendor if it's specifically sold by CARL (also because it makes no sense for it to be able to restock any arbitrary clothing vendor)